LukeberryPi/posso-perguntar
# Replace twitter picture

Because Twitter images are unreliable — their URLs change, and it's a pain to keep track of them — this PR replaces the profile picture with a hosted image.

Closes #3 

## How to test

1. Open local server
2. See image working properly
